tests/diviner/test_diviner_model_export.py

```diff
@@ -79,6 +79,17 @@ def diviner_custom_env(tmp_path):
     return conda_env
 
 
+@pytest.fixture
+def diviner_groups(diviner_data):
+    groups = []
+    for i in [0, -1]:
+        key_entries = []
+        for value in diviner_data.df[diviner_data.key_columns].iloc[[i]].to_dict().values():
+            key_entries.append(list(value.values())[0])
+        groups.append(tuple(key_entries))
+    return groups
+
```

Comment:
Can we move this change to a separate PR?

Reply:
Not directly necessary but this is for convenience to use the fixture in the docker flavor tests.